portfolio/src/main/java/com/google/sps/servlets/LoadCommentsServlet.java

```diff
@@ -34,26 +34,66 @@
 @WebServlet("/load-comments")
 public class LoadCommentsServlet extends HttpServlet {
 
+  int maximumComments = 10;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
+    int count = 0;
     List<Comment> comments = new ArrayList<>();
+    comments.add(new Comment(0, Integer.toString(results.countEntities()), 0));
     for (Entity entity : results.asIterable()) {
+      if (count == maximumComments) {
```

Comment:
why not `if results.countEntities > maximumComments` outside of loop?

Reply:
Will change to results.countEntities!

Reply:
done! I converted results to a list and used the size() method, the list made me able to iterate through the array by index and so I just got rid of the for each loop and used a for loop.